### PR TITLE
chore: bump semantic versions to 1.0.1

### PR DESCRIPTION
**Pull request by aquigle**
Bumps the semantic versions everywhere after the latest dependency upgrades

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

**Comment by honglu** [comment]
### AWS CodeBuild CI Report

* Result: SUCCEEDED
* [Build Logs](https://663jj0k3ya.execute-api.us-east-1.amazonaws.com/Prod/buildlogs?key=2d4280f3-a9b7-4744-9064-ea3d592a4898%2Fbuild.log) (available for 30 days)

*Powered by [github-codebuild-logs](https://serverlessrepo.aws.amazon.com/applications/arn:aws:serverlessrepo:us-east-1:277187709615:applications~github-codebuild-logs), available on the [AWS Serverless Application Repository](https://aws.amazon.com/serverless/serverlessrepo/)*

File: analytics/sam/app/template.yaml [hunk 1]
```diff
@@ -29,7 +29,7 @@ Resources:
     Properties:
       Location:
         ApplicationId: arn:aws:serverlessrepo:us-east-1:646794253159:applications/aws-dynamodb-stream-eventbridge-fanout
-        SemanticVersion: 1.0.0
+        SemanticVersion: 1.0.1
```
Comment (jlhood) [inline analytics/sam/app/template.yaml:32]: We should merge [the PR](https://github.com/awslabs/aws-dynamodb-stream-eventbridge-fanout/pull/15) in this repo and upgrade it to 1.0.2 before updating this.

File: analytics/sam/cicd/template.yaml [hunk 2]
```diff
@@ -26,7 +26,7 @@ Resources:
     Properties:
       Location:
         ApplicationId: arn:aws:serverlessrepo:us-east-1:646794253159:applications/aws-sam-codepipeline-cd
-        SemanticVersion: 0.1.3
+        SemanticVersion: 0.1.4
```
Comment (jlhood) [inline analytics/sam/cicd/template.yaml:29]: Why this change? That semver doesn't exist... You should only be bumping semvers in the template metadata section as well as any `AWS::Serverless::Application`s that reference apps whose MVs you're bumping in this PR. Same feedback for all places where you're bumping the version on aws-sam-codepipeline-cd.

```text
% aws serverlessrepo list-application-versions --application-id arn:aws:serverlessrepo:us-east-1:646794253159:applications/aws-sam-codepipeline-cd
{
    "Versions": [
        {
            "ApplicationId": "arn:aws:serverlessrepo:us-east-1:646794253159:applications/aws-sam-codepipeline-cd",
            "CreationTime": "2019-09-04T23:50:57.809Z",
            "SemanticVersion": "0.1.0",
            "SourceCodeUrl": "https://github.com/awslabs/aws-sam-codepipeline-cd/tree/0.1.0"
        },
        {
            "ApplicationId": "arn:aws:serverlessrepo:us-east-1:646794253159:applications/aws-sam-codepipeline-cd",
            "CreationTime": "2019-09-04T23:54:00.997Z",
            "SemanticVersion": "0.1.1",
            "SourceCodeUrl": "https://github.com/awslabs/aws-sam-codepipeline-cd/tree/0.1.1"
        },
        {
            "ApplicationId": "arn:aws:serverlessrepo:us-east-1:646794253159:applications/aws-sam-codepipeline-cd",
            "CreationTime": "2019-09-16T19:26:49.995Z",
            "SemanticVersion": "0.1.3",
            "SourceCodeUrl": "https://github.com/awslabs/aws-sam-codepipeline-cd/tree/0.1.3"
        },
        {
            "ApplicationId": "arn:aws:serverlessrepo:us-east-1:646794253159:applications/aws-sam-codepipeline-cd",
            "CreationTime": "2019-10-14T01:17:47.607Z",
            "SemanticVersion": "1.0.0",
            "SourceCodeUrl": "https://github.com/awslabs/aws-sam-codepipeline-cd/tree/1.0.0"
        },
        {
            "ApplicationId": "arn:aws:serverlessrepo:us-east-1:646794253159:applications/aws-sam-codepipeline-cd",
            "CreationTime": "2019-10-25T19:11:46.577Z",
            "SemanticVersion": "1.1.0",
            "SourceCodeUrl": "https://github.com/awslabs/aws-sam-codepipeline-cd/tree/1.1.0"
        }
    ]
}
```

File: static-website/sam/app/template.yaml [hunk 3]
```diff
@@ -35,7 +35,7 @@ Resources:
     Properties:
       Location:
         ApplicationId: arn:aws:serverlessrepo:us-east-1:375983427419:applications/deploy-to-s3
-        SemanticVersion: 2.2.0
+        SemanticVersion: 2.2.1
```
Comment (jlhood) [inline static-website/sam/app/template.yaml:38]: That semver doesn't exist. Should revert this change.

**Review by jlhood — CHANGES_REQUESTED** [review]

File: analytics/sam/app/template.yaml (hunk 1 above)
Comment (jlhood) [reply analytics/sam/app/template.yaml]: I went ahead and merged that PR and updated the app to 1.0.2 so you can change this to reference 1.0.2 now.

**Comment by honglu** [comment]
### AWS CodeBuild CI Report

* Result: SUCCEEDED
* [Build Logs](https://663jj0k3ya.execute-api.us-east-1.amazonaws.com/Prod/buildlogs?key=81a87d60-6a75-41ab-a74d-2ae5b15beafa%2Fbuild.log) (available for 30 days)

*Powered by [github-codebuild-logs](https://serverlessrepo.aws.amazon.com/applications/arn:aws:serverlessrepo:us-east-1:277187709615:applications~github-codebuild-logs), available on the [AWS Serverless Application Repository](https://aws.amazon.com/serverless/serverlessrepo/)*

**Comment by honglu** [comment]
### AWS CodeBuild CI Report

* Result: SUCCEEDED
* [Build Logs](https://663jj0k3ya.execute-api.us-east-1.amazonaws.com/Prod/buildlogs?key=28d6a1d9-64b4-49fc-b937-f3b1c4691ebc%2Fbuild.log) (available for 30 days)

*Powered by [github-codebuild-logs](https://serverlessrepo.aws.amazon.com/applications/arn:aws:serverlessrepo:us-east-1:277187709615:applications~github-codebuild-logs), available on the [AWS Serverless Application Repository](https://aws.amazon.com/serverless/serverlessrepo/)*

**Review by jlhood — APPROVED** [review]
